<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="DESKTOP-RL7I7P6" name="com.admin.qa.testcases.AddUserPageTest" tests="5" failures="0" timestamp="05 Aug 2019 16:19:34 GMT" time="8.680" errors="1">
  <testcase name="addUserPageSetup" time="8.680" classname="com.admin.qa.testcases.AddUserPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.admin.qa.pages.UserPage.verifyadduser(UserPage.java:15)
	at com.admin.qa.testcases.AddUserPageTest.addUserPageSetup(AddUserPageTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </error>
  </testcase> <!-- addUserPageSetup -->
  <testcase name="verifyAddUserPageTitle" time="0.000" classname="com.admin.qa.testcases.AddUserPageTest">
    <skipped/>
  </testcase> <!-- verifyAddUserPageTitle -->
  <testcase name="verifyAddUserPageLabel" time="0.000" classname="com.admin.qa.testcases.AddUserPageTest">
    <skipped/>
  </testcase> <!-- verifyAddUserPageLabel -->
  <testcase name="verifyAddUserPageFormLables" time="0.000" classname="com.admin.qa.testcases.AddUserPageTest">
    <skipped/>
  </testcase> <!-- verifyAddUserPageFormLables -->
  <testcase name="verifyAddUserForm" time="0.000" classname="com.admin.qa.testcases.AddUserPageTest">
    <skipped/>
  </testcase> <!-- verifyAddUserForm -->
</testsuite> <!-- com.admin.qa.testcases.AddUserPageTest -->
